solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/AddressView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AddressView.swift
+//  DeliveryApp
+//
+//  Created by Alexandre Cardoso on 21/02/22.
+//
+
+import UIKit
+
+protocol AddressViewDelegate: AnyObject {
+    func didEditAddress()
+}
+
+final class AddressView: UIView {
+    
+    private lazy var addressLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "R. Guiratinga, 500"
```

Comment:
Considerando que esse nosso componente vai ser configurado através do método `updateView`, não precisamos setar o texto do endereço diretamente. Podemos talvez utilizar um placeholder:
```suggestion
        label.text = "Selecione um Endereço"
```